frontend/src/components/dashboard/create/email/EmailDetail.tsx

```diff
@@ -94,14 +98,20 @@ const EmailDetail = () => {
           from={emailCampaign.from}
         />
         <div className="separator"></div>
-        {stats.status && (
+        {stats.status && stats.status !== Status.Scheduled && (
```

Comment:
shouldn't we just use `campaign.status` here instead of forcing `status` to be returned in the get stats endpoint?

Reply:
fixed

Reply:
status was previously returned in stats endpoint already, but have changed to refer to campaign status instead.